### PR TITLE
Day 1 game label/create classes

### PR DESCRIPTION
**Pull request by sonyco-4u**
**Checklist before requesting a review**

-  I have performed a self-review of my code
- Create a Game class in a separate .rb file.
- Create an Author class with an association to the Item class (in a separate .rb file).

 Implement methods:
🔢 💯 🥇

### CODE REVIEW

File: src/classes/game.rb [hunk 1]
```diff
@@ -1 +1,17 @@
-# Game class
+# Game class
+require_relative './item'
+require 'date'
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @id = rand(1..1000)
```
Comment (Camilovelag) [inline src/classes/game.rb]: Kindly don't override `@id` value initialization because you have already done this process in the Item class.

**Review by Camilovelag — CHANGES_REQUESTED** [review]
Hi @sonyco-4u 👋 
Your PR is almost completed. I have some changes to be done:

**Review by luccamicale — APPROVED** [review]
Hi @sonyco-4u I checked your code. It is very tidy and clean. Great Job.

👍 Good Linters
💯 Perfect Best Practices

**Review by Camilovelag — APPROVED** [review]
Good job. Time to merge 👍

**Pull request merged**